pkgs/os-specific/linux/usbip/default.nix

```
@@ -27,7 +27,7 @@ stdenv.mkDerivation {
   meta = with lib; {
     homepage = "https://github.com/torvalds/linux/tree/master/tools/usb/usbip";
     description = "allows to pass USB device from server to client over the network";
-    license = licenses.gpl2;
+    license = licenses.gpl2Only;
```

The file header says [`GPL-2.0-or-later`](https://github.com/torvalds/linux/blob/master/tools/usb/usbip/src/usbipd.c#L1).

```suggestion
    license = licenses.gpl2Plus;
```

---

But some other files in the linux/tools/usb/usbip tree have a GPL-2.0 identifier, like here: https://github.com/torvalds/linux/blob/master/tools/usb/usbip/libsrc/list.h or https://github.com/torvalds/linux/blob/master/tools/usb/usbip/cleanup.sh.

---

Then it looks like a dual-licensed scenario: `with licenses; [ gpl2Only gpl2Plus ]`

---

Thank you for noticing. I implemented your suggestion.